miso-web/src/main/webapp/pages/editLibrary.jsp

```diff
@@ -342,13 +342,21 @@
   <td><form:input id="dnaSize" path="dnaSize"/></td>
 </tr>
 <tr>
-  <td>Volume<span id="volumeUnits"></span>:</td>
+  <td>Volume<span id="volumeUnitsDisplay"></span>:</td>
```

Comment:
No longer needs displayed since it's shown (and possibly changed) below. Same with concentrationUnits, and on other pages